packages/landing/src/components/new.feature.tsx

```diff
@@ -0,0 +1,48 @@
+import { Component, ReactNode } from 'react';
+
+import { Config } from '../config.js';
+import { WindowUrl } from '../url.js';
+import { FeatureUpdates } from './feature.updates.js';
+
+const baseUrl = WindowUrl.baseUrl();
+
+/**
+ * Please updated the following settings and descriptions for new features pop up
+ */
+const bigImage = new URL('assets/Lg+3D+Maps+splash.gif', baseUrl).href; // Large gif file location
```

Comment:
could this be turned into a new.feature.3dmap.tsx or similar ?

so we have a component per new feature

Reply:
Updated.